default.yaml

```diff
@@ -0,0 +1,10 @@
+splunk:
+  conf:
+    - key: indexes
+      value:
+        directory: /opt/splunk/etc/system/local
+        content:
+          $_index_name:
```

Comment:
so do we need both this and indexes.conf?

Reply:
No. I was experimenting with this probably remove it later.